evalml/tests/automl_tests/dask_testing.py

```diff
@@ -0,0 +1,102 @@
+import time
+
+from evalml.automl.utils import AutoMLData
+from evalml.exceptions import PipelineScoreError
+from evalml.objectives.utils import get_objective
+from evalml.pipelines import BinaryClassificationPipeline
+from evalml.preprocessing.data_splitters import TrainingValidationSplit
+
+
+# Top-level replacement for AutoML object to supply data for testing purposes.
+def err_call(*args, **kwargs):
+    """No-op"""
+
+
+ensembling_indices = [0]
+data_splitter = TrainingValidationSplit()
+problem_type = "binary"
+objective = get_objective("Log Loss Binary", return_instance=True)
+additional_objectives = []
+optimize_thresholds = False
+error_callback = err_call
+random_seed = 0
+automl_data = AutoMLData(ensembling_indices=ensembling_indices,
+                         data_splitter=data_splitter,
+                         problem_type=problem_type,
+                         objective=objective,
+                         additional_objectives=additional_objectives,
+                         optimize_thresholds=optimize_thresholds,
+                         error_callback=error_callback,
+                         random_seed=random_seed)
```

Comment:
To avoid strange errors arising from accidentally mutating global state: could you please create this stuff in a test fixture, which runs once per test, instead of once at import-time as things stand currently?

You could even move it to [evalml/tests/conftest.py](https://github.com/alteryx/evalml/blob/main/evalml/tests/conftest.py).